### DISCUSSION TITLE
Managing presets over WiFi

### DISCUSSION THREAD

**Discussion by zsoltmar** [probonopd/MiniDexed#364] [Q&A]
Would be nice to have an AP mode to be able to edit presets over the network. Is anyone doing any work in this direction? Even an ssh connection would be a great improvement over having to fish out the SD card every time there's a need to make adjustments... Also, any way to extend the banks limit from 128? Cheers!

**Comment by probonopd** [comment]
I don't think ssh has been implemented in https://github.com/rsta2/circle which MiniDexed is using.

**Comment by zsoltmar** [comment]
Will look into this in a bit, maybe there's some way to get this in there? https://iosoft.blog/zerowi/
